src/layer/vector/Renderer.js

```diff
@@ -67,7 +67,8 @@ export var Renderer = Layer.extend({
 			viewreset: this._reset,
 			zoom: this._onZoom,
 			moveend: this._update,
-			zoomend: this._onZoomEnd
+			move: this._update,
```

Comment:
This is maybe not good because it can cause performance problems

Reply:
Will need to spend some time understanding the performance implications here, but otherwise this is great!